### ISSUE TITLE
projected ParticleContainer?

### ISSUE THREAD

**Issue by kimsyv** [pixijs/pixi-projection#11]
Would it be possible to add ParticleContainer2d/PartcleContainer2s to this plugin?

Doing convertTo2d() on a ParticleContainer and then adding a Sprite2d to it results in this:

```
ParticleRenderer.js:147 Uncaught TypeError: Cannot read property 'prepend' of undefined
    at ParticleRenderer.render (ParticleRenderer.js:147)
    at ParticleContainer.renderWebGL (ParticleContainer.js:214)
    at ReelSymbol.renderWebGL (Container.js:414)
    at Container2d.renderWebGL (Container.js:414)
    at Reel.renderWebGL (Container.js:414)
    at Reels.renderWebGL (Container.js:414)
    at Container.renderAdvancedWebGL (Container.js:467)
    at Container.renderWebGL (Container.js:405)
    at WebGLRenderer.render (WebGLRenderer.js:329)
```

**Comment by ivanpopelyshev** [comment]
No, I haven't made ParticleContainer mod yet. Are you sure that you need particles? How many sprites are there?

**Comment by kimsyv** [comment]
No, I am not sure I need particles. I am just toying with the idea to contain most of my projection sprites within a ParticleContainer - mostly to improve framerate on a slow android tablet.
But we are also thinking of adding some particles to our game, but that can perhaps then be done without projection.

**Comment by ivanpopelyshev** [comment]
How many sprites do you need to be in that container? If its below 100 then its not a case for PatricleContainer, I'm sure.

I implemented TilingSprite, and I think next step will be full 3d support. However, if you really need particles, I can implement them before 3d.

**Comment by kimsyv** [comment]
Well, there is no rush with this. I managed to get the framerate on a slow android tablet to be acceptable by optimizing heavily overall. But the number of sprites is mostly around 100 in my main projected container. Each sprite can sometimes have around 40 child sprites - not all at the same time, though. So there can be times where I have 500 - 1000 sprites in my main projected container.

Do the full 3d support first. That sounds interesting :)
